### PR TITLE
Add initial github pages content

### PR DESCRIPTION
**Pull request by mayaCostantini**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+No need to be an expert on Django, everyone is welcome.
+
+
+Bonjour à tous,
+
+Ce groupe à pour but de réunir les djangonautes autour de Paris afin d'échanger et partager des connaissances et bien sur, se rencontrer !
+
+Pas besoin d'être un expert en Django, tout le monde est le bienvenu.
+
+## Code of conduct
```
Comment (sabderemane) [inline index.md:27]: May be add a section for links like Twitter and so on ?

File: index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The purpose of the group is to gather djangonauts around Paris to exchange and share knowledge, and of course, meet each other !
+
+No need to be an expert on Django, everyone is welcome.
+
+
+Bonjour à tous,
+
+Ce groupe à pour but de réunir les djangonautes autour de Paris afin d'échanger et partager des connaissances et bien sur, se rencontrer !
+
+Pas besoin d'être un expert en Django, tout le monde est le bienvenu.
```
Comment (sabderemane) [inline index.md:20]: Should we translate everything? I don't know if it's will be good to have a link with all pages translated in one language, what do you think?

**Review by sabderemane — COMMENTED** [review]
Only few topics to review but sounds good to me 😄

File: index.md (hunk 2 above)
Comment (mayaCostantini) [reply index.md:20]: Right, maybe we can leave it as is for now as we do not have much content and add different links with the EN/FR version later?

**Review by sabderemane — APPROVED** [review]

**Pull request merged**